src/components/searchSection/CategoryList.js

```diff
@@ -6,17 +6,11 @@ const MAXIMUM_TABS_TO_SHOW = 9;
 
 function CategoryList({ items }) {
   const [activeBubble, setActiveBubble] = useState(0);
-  const [category, setCategory] = useState('');
+  const [setCategory] = useState('');
```

Comment:
this will give you error

```suggestion
  const [category, setCategory] = useState('');
```